src/logic/setup.py

```diff
@@ -41,17 +34,16 @@
     HISTORY = f.read()
 
 setup(
-    name='logic',
+    name='logicab',
```

Comment:
I'm curious why it was renamed as `logicab`?

Reply:
Wrong writing, fixed